### DISCUSSION TITLE
clean stale, unused branches

### DISCUSSION THREAD

**Discussion by zhaozg** [luvit/luv#643] [General]
There are a lots of branch to fix issues or try new features, but they too old, and I think it is unused anymore.

I have remove some merged branches, and others I can't make up mind.

So, we need do somethings, branch owners please delete them if unused, or do discuss for that.

| branch |  safe to delete | vote |
|--------|---------------|------|
| coro-error|     |   |
| error-redo|     |   |
| fix-luvi|     |   |
| lightuserdata|     |   |
| schema|     |   |

[schema](https://github.com/luvit/luv/tree/schema) need talk about.

zhaozg +1 for deletable branchs.

Keep listening voice from @luvit/luvit-admin and contributors.

@Bilal2453 give options, and I narrow it.  delete [uv-update-1.0.0](https://github.com/luvit/luv/tree/uv-update-1.0.0),[upstream-cwd-fix](https://github.com/luvit/luv/tree/upstream-cwd-fix), [luvi-build](https://github.com/luvit/luv/tree/luvi-build) are safe, see [comment](https://github.com/luvit/luv/issues/642#issuecomment-1493303152)

**Comment by Bilal2453** [comment]
All current branches are safe for deletion as they have been merged except for 6: [lightuserdata](https://github.com/luvit/luv/tree/lightuserdata), [coro-error](https://github.com/luvit/luv/tree/coro-error), [uv-update-1.0.0](https://github.com/luvit/luv/tree/uv-update-1.0.0), [upstream-cwd-fix](https://github.com/luvit/luv/tree/upstream-cwd-fix), [luvi-build](https://github.com/luvit/luv/tree/luvi-build), [fix-luvi](https://github.com/luvit/luv/tree/fix-luvi).  If that helps.

note: I looked at fix-luvi, which updates luajit and libuv to a currently outdated version, removes lua-compat-5.3, and manually does Lua 5.2/5.3 compatibility checks (for a couple of things). Doesn't seem applicable in current luvi/luv.

**Comment by zhaozg** [comment]
[uv-update-1.0.0](https://github.com/luvit/luv/tree/uv-update-1.0.0) should be delete, now we already libuv v1.x.  
[upstream-cwd-fix](https://github.com/luvit/luv/tree/upstream-cwd-fix) should be delete, which already solved on 2014-10-17, upstream libuv bug.
[luvi-build](https://github.com/luvit/luv/tree/luvi-build) should be delete, which already merged on 2018-04-24 by Tim Caswell.
